.drone.star

```diff
@@ -948,19 +949,28 @@ def wopiValidatorTests(ctx, storage, accounts_hash_difficulty = 4):
                  [
                      {
                          "name": "wopiserver",
-                         "image": "cs3org/wopiserver:v10.3.0",
```

Comment:
Is it intended to fully replace the cs3 wopi server as of now?
From my understanding it is still used in e.g. bysc - we maybe still want to run ci against it together with the new collab server ...

Reply:
I'll check if I can copy the testsuite to run both

Reply:
@individual-it any hint on what is the right way to do it? I could duplicate the `wopiValidatorTests` and adjust the steps in order to run the tests against the cs3 wopi server...

Reply:
a new pipeline stage has been added to run the wopi validator with the cs3 wopi server